.github/workflows/release.yaml

```
@@ -5,6 +5,20 @@ on:
     types: [created]
 
 jobs:
+  update_readme:
+    name: Update Readme
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - run: |
+        sed -r -i "s/aws-nuke:v[0-9]+\.[0-9]+\.[0-9]+/aws-nuke:${GITHUB_REF#refs/tags/}/" README.md
```

```suggestion
        sed -r -i "s/aws-nuke:v[0-9]+\.[0-9]+\.[0-9]+/aws-nuke:${GITHUB_REF#refs/tags/}/g" README.md
```

Then you can delete the second line.

---

No, the diff is `aws-nuke**:**v` and `aws-nuke**-**v`

---

hmkay